Same Padding:

Output_height = Input_height/Strides

Output_width = Input_width/Strides

Valid Padding:

Output_height = (Input_height - kernel_height +1 ) / Strides

Output_width = (Input_width - kernel_width +1 ) / Strides

In [1]:
from tensorflow import keras

In [2]:
inputs = keras.Input((28,28,4))

In [3]:
outputs = keras.layers.Conv2D(32, kernel_size=3, strides=(1,1), padding='valid',
                              use_bias=True)(inputs)

In [4]:
model = keras.Model(inputs, outputs)

In [5]:
model.summary()

Model: "model"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
input_1 (InputLayer)         [(None, 28, 28, 4)]       0         
_________________________________________________________________
conv2d (Conv2D)              (None, 26, 26, 32)        1184      
Total params: 1,184
Trainable params: 1,184
Non-trainable params: 0
_________________________________________________________________


Parameters = (Input_channel * kernel_height * kernel_width * Bais) * kernel_numbers

1184 = (4*3*3*1)*32  if has Bias = 1, else Bias = 0